### ISSUE TITLE
evebox: change URL in /etc/scirius/local_settings.py

### ISSUE THREAD

**Issue by jasonish** [StamusNetworks/SELKS#50]
With PR #49 evebox is now bound to localhost.  The following change is required in /etc/scirius/local_settings.py but I could not find the original source for this:

```
diff -u /etc/scirius/local_settings.py.orig etc/scirius/local_settings.py
--- /etc/scirius/local_settings.py.orig	2016-07-11 10:22:26.466937756 -0500
+++ etc/scirius/local_settings.py	2016-07-11 10:22:33.950948651 -0500
@@ -31,7 +31,7 @@
 #SURICATA_UNIX_SOCKET = "/var/run/suricata/suricata-command.socket"
 
 USE_EVEBOX = True
-EVEBOX_ADDRESS = "selks:5636"
+EVEBOX_ADDRESS = "localhost:5636"
 
 USE_SURICATA_STATS = True
 USE_LOGSTASH_STATS = True
```

This also prevents being taken to the wrong SELKS instance in the case of issue #48.

**Comment by pevma** [comment]
This is part of the Scirius packaging. I will adjust that.

**Comment by adescamps11** [comment]
I've had the same problem. 
Because of changing the hostname, I couldn't start the evebox's service so I had to change the hostname in /etc/systemd/system/evebox.service

> [Service]
ExecStart=/usr/bin/evebox -e http://localhost:9200 --host SELKSCLI
#SELSKCLI is the hostname I changed

Besides, when I tried to start the evebox's service after that change, the system recommended me to use the `systemctl daemon-reload` command

> Warning: Unit file of evebox.service changed on disk, 'systemctl daemon-reload' recommended.

After that, all worked nicely.

**Comment by jasonish** [comment]
Fixed in latest 3.0 branch.  Tested access to Evebox from the VM itself, and from another machine.
